client/src/app/api/auth/google/route.ts

```diff
@@ -25,6 +25,7 @@ export async function POST(req: NextRequest) {
 
     if (response.ok) {
       const { accessToken } = await response.json();
+      //console.log("accessToken", accessToken);
```

Comment:
It’s not necessarily for future use, but it can make things easier, especially when using Postman, as we can quickly access the accessToken by simply copying and pasting it from the console